packages/hdwallet-gridplus/src/ethereum.ts

```diff
@@ -1,8 +1,8 @@
 import { Common, Hardfork } from "@ethereumjs/common";
-import { TransactionFactory, TypedTxData } from "@ethereumjs/tx";
+import { RLP } from "@ethereumjs/rlp";
+import { TransactionFactory, TransactionType, TypedTxData } from "@ethereumjs/tx";
```

Comment:
This works in CI in PR because CI only runs `yarn && yarn build`  but if you try and publish (i.e start from fresh master and built packages, checkout this branch, build packages, and publish which will remove `node_modules`) this will fail:

<img width="1208" height="110" alt="Image" src="https://github.com/user-attachments/assets/6afc9b37-b2dc-4cd2-bdb4-8d043ae7b041" />
<img width="1378" height="103" alt="Image" src="https://github.com/user-attachments/assets/c45079d9-c9e6-4f41-ad98-65f7f21bc656" />

Reply:
Making sense now with the addition of nohoist package for gridplus. Fixing.